readthedocs/oauth/migrations/0011_add_remote_relation_model.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:10
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('oauth', '0010_index_full_name'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
```

Comment:
I never used this so I will need some time to process how it works. The documentation on 2.2 is not enough, but I found that [3.1 docs](https://docs.djangoproject.com/en/3.1/ref/migration-operations/#separatedatabaseandstate) has an example of what we want to do exactly: https://docs.djangoproject.com/en/3.1/howto/writing-migrations/#changing-a-manytomanyfield-to-use-a-through-model

Reply:
It would have been a lot simpler if I had seen the example but What we are doing here is quite similar.